http.go

```diff
@@ -280,18 +281,25 @@ func (c *httpFS) send(httpClient *http.Client, req *http.Request) (*http.Respons
 // access fs. It should be accessed by clients created using this
 // package's HTTP func.
 func HTTPHandler(fs FileSystem, logTo io.Writer) http.Handler {
+	return HTTPHandlerWithDelay(fs, logTo, 0)
+}
+
+func HTTPHandlerWithDelay(fs FileSystem, logTo io.Writer, delay time.Duration) http.Handler {
 	if logTo == nil {
 		logTo = ioutil.Discard
 	}
-	return &httpFSHandler{fs, log.New(logTo, "rwvfs HTTP: ", log.Flags())}
+	return &httpFSHandler{fs, log.New(logTo, "rwvfs HTTP: ", log.Flags()), delay}
 }
 
 type httpFSHandler struct {
-	fs  FileSystem
-	log *log.Logger
+	fs    FileSystem
+	log   *log.Logger
+	delay time.Duration
 }
 
 func (h *httpFSHandler) ServeHTTPAndReturnError(w http.ResponseWriter, r *http.Request) error {
+	time.Sleep(h.delay)
```

Comment:
Sounds good, thanks for explaining!

I recommend adding a comment, no matter how small, to `ServeHTTPAndReturnError` to make it clear that it being exported is not an unintentional typo but an intended action.

```Go
// ServeHTTPAndReturnError serves an HTTP request and returns an error.
```

Otherwise, at a glance, it's hard to tell if `ServeHTTPAndReturnError` used externally or not. To quote go lint:

> exported method ServeHTTPAndReturnError should have comment or be unexported